certdocs/0-DDLN-212-SDD.md

```diff
@@ -167,61 +167,110 @@ The RMT SHALL check that the requirements defined in each document have a correc
 
 ###### Attributes:
 - Rationale: this helps ensure that no requirement sequence numbers are accidentally skipped.
 - Verification: Unit test.
 - Safety impact: None.
-- Parent: REQ-0-DDLN-SWH-003
+- Parent: REQ-RQTQ-SWH-003
 
-##### REQ-0-DDLN-SWL-017 Deleted requirements.
+##### REQ-RQTQ-SWL-017 Deleted requirements.
 
-Deleted requirements are requirements that do not apply anymore (e.g. they are obsolete). Deleted requirements SHALL be marked by changing the title to "Deleted", for example "REQ-0-DDLN-SWL-015 Deleted"
+Deleted requirements are requirements that do not apply anymore (e.g. they are obsolete). Deleted requirements SHALL be marked by changing the title to "Deleted", for example "REQ-RQTQ-SWL-015 Deleted"
 
 Deleted requirements SHALL not be checked for completeness and all the tasks associated with them SHALL be closed as WONTFIX. All references to a deleted requirement SHALL be marked as errors.
 
 ###### Attributes:
 - Rationale: continuous requirement numbering helps ensure that no requirements were accidentally skipped and completely deleting requirements would create gaps in the numbering.
 - Verification: Unit test.
 - Safety impact: None.
-- Parent: REQ-0-DDLN-SWH-003
+- Parent: REQ-RQTQ-SWH-003
 
-##### REQ-0-DDLN-SWL-015 Data structure for keeping requirements and their hierarchy
+##### REQ-RQTQ-SWL-015 Data structure for keeping requirements and their hierarchy
 
 The interface between the parsing tool and the report generation tool SHALL be a data structure that maps requirement IDs to a requirement structure. The requirement structure will hold all the data about the requirement that is needed for the report generation (ID, body, attributes, parents, children, etc.). The data structure is built by traversing the entire git repository and parsing all files that may contain or reference requirements, such as `.lyx`/`.md` requirement files and `.cc`/`.hh` source files).
 
 ###### Attributes:
 - Rationale: this data structure will be used for report generation and graph verification.
 - Verification: Unit test
 - Safety impact: None
-- Parents: REQ-0-DDLN-SWH-004, REQ-0-DDLN-SWH-005
+- Parents: REQ-RQTQ-SWH-004, REQ-RQTQ-SWH-005
 
-##### REQ-0-DDLN-SWL-006. Tracing system to high, low level, implementation, test.
+##### REQ-RQTQ-SWL-020 Requirements document organization
 
-The RMT SHALL, given a list of requirements given to or generated by the project as checked in to the repository, be able to generate parent and child requirements and code ordered from system to high level to low level requirement to implementation and test, including missing continuations.
+Requirements documents SHALL be organized as follows:
+  - requirements documents for a specific project SHALL be located in the `certdocs` top-level directory for that particular project
+  - for projects that are part of a larger system, the system level requirements of the project will have parents in the system level requirements of the larger system
```

Comment:
I started writing an email on this topic. I'll paste here: 

> If we end up adopting reqtraq, it will have to exist alongside another requirements tool which will contain system level requirements that reqtraq will trace up to. So we'll need a way to trace to requirements that reqtraq is unaware of without raising an error (perhaps a boolean attribute in the requirement that should instruct reqtraq to permit this).